configs/components/rubygem-ffi.rb

```diff
@@ -0,0 +1,35 @@
+component "rubygem-ffi" do |pkg, settings, platform|
+  pkg.version "1.9.6"
+
+  if platform.is_windows?
+    if platform.architecture == "x64"
+      pkg.md5sum "daf0d310a5c498906f94f8c10006fd39"
+      pkg.url "http://buildsources.delivery.puppetlabs.net/ffi-#{pkg.get_version}-x64-mingw32.gem"
+    else
+      pkg.md5sum "5a0b3f3602f65a2b7fe6569a3333a068"
+      pkg.url "http://buildsources.delivery.puppetlabs.net/ffi-#{pkg.get_version}-x86-mingw32.gem"
```

Comment:
It's where we're installing all our sources from, including other gems (https://github.com/puppetlabs/puppet-agent/blob/master/configs/components/rubygem-deep-merge.rb#L4). We might switch this over later, but I'm not sure. I don't know the long term plan with these. I know Mike's been looking at setting up a rubygem server on nexus